.github/workflows/lint.yml

```
@@ -57,6 +57,16 @@ jobs:
       - name: Validate Podspec
         run: ./scripts/pod-lib-lint.sh ${{ matrix.platform }} ${{ matrix.podspec}} ${{ matrix.library_type}}
 
+  lint-hybridsdk-podspec:
+    name: pod lint Sentry/HybridSDK
+    runs-on: macos-12
```

`m`: macos-13 is way faster.
```suggestion
    runs-on: macos-13
```